ome/omero-figure
# Reverse intensity

This adds support for Reverse intensity to figure - see https://trello.com/c/11hiGrmx/13-reverse-intensity-codomain

We also update the ```luts_10.png``` with the Janelia LUTs. NB: this is still hard-coded since changing that is more work and will mean that OMERO.figure will rely on https://github.com/openmicroscopy/openmicroscopy/pull/5318 (OMERO 5.3.3 at least).
Can do this in a future PR/release.

This also moves the channel-toggle buttons down beside the sliders, to align OMERO.figure with our other clients. This frees up more space for other buttons beside the image, such as 'flip'. cc @carandraug.

Includes a fix for toggling channels bug: https://trello.com/c/yWFPR9MJ/160-bug-toggle-channels

![screen shot 2017-06-13 at 15 21 30](https://user-images.githubusercontent.com/900055/27087462-0425ae88-504d-11e7-97d2-1feed44afd55.png)

To test:
 - Import images into figure, including some with channels that are saved with reverseIntensity.
 - Should appear same in figure (with reverse intensity enabled where appropriate).
 - Sliders should have gradient for plain colour channels (no LUT)
 - LUT and colour gradient should be reversed with reverse intensity
 - If multiple images are selected, channel should be displayed as 'reverse intensity' only if channel is reversed in ALL images.
 - Color-picker dropdown option 'Reverse Intensity' should be checked if all selected images are reversed.
 - Clicking the 'Reverse Intensity' option should turn it on/off.
 - Layout should look OK as below and other channel controls and rotation/z-projection should be unaffected.
 - Export of figure should respect reverse intensity settings.
